home/forms/professor_forms.py

```diff
@@ -205,7 +205,7 @@ class ProfessorFormReview(ProfessorForm):
     def __init__(self, user, professor, **kwargs):
         super().__init__(user, Review.ReviewType.REVIEW, **kwargs)
 
-        courses = professor.course_set.order_by("name").distinct()
+        courses = professor.course_set.order_by("name")
```

Comment:
this is reverting https://github.com/planetterp/PlanetTerp/pull/41

Reply:
this is still reverting #41 as far as I can tell. Same with admin.py - the change with adding the `rating `embed field is being reverted there.